Cabal/src/Distribution/Verbosity.hs

```diff
@@ -330,15 +333,15 @@ isVerboseMarkOutput = isVerboseFlag VMarkOutput
 isVerboseNoWrap :: Verbosity -> Bool
 isVerboseNoWrap = isVerboseFlag VNoWrap
 
--- | Test if we had called 'lessVerbose' on the verbosity
+-- | Test if we had called 'lessVerbose' on the verbosity.
 isVerboseQuiet :: Verbosity -> Bool
 isVerboseQuiet = vQuiet
 
 -- | Test if we should output timestamps when we log.
 isVerboseTimestamp :: Verbosity -> Bool
 isVerboseTimestamp = isVerboseFlag VTimestamp
 
--- | Test if we should output to stderr when we log.
+-- | Test if we should output to 'stderr' when we log.
```

Comment:
Is stderr really visible here? I haven't checked, but if one generates haddocks, this should be visible.

Reply:
It is not linked, but still teletyped.  
Imports for haddock's sake are probably not advised.

Reply:
Yes, indeed. Perhaps with a module prefix it would work, but not worth it, either, IMHO.

Good job on haddock's part that it renders it fine despite the warning (or error?). Good to know.